notebooks/basics/quantum-circuits.ipynb

```diff
@@ -263,24 +269,24 @@
     "$$\n",
     "\n",
     "<ol style=\"list-style-type:none\"><li>\n",
-    "Note that the identity matrix appears on the left-hand side of the tensor product and $H$ appears on the right-hand side because that is the ordering that is consistent with the rule described above.\n",
+    "Note that the identity matrix is on the left of the tensor product and $H$ is on the right. This is consistent with Qiskit's qubit ordering.\n",
     "</ol>\n",
     "\n",
     "<ol start=\"2\"><li>\n",
-    "      The second operation that is performed is the controlled-NOT operation, where $\\mathsf{Y}$ is the control \n",
+    "      The second operation  is the controlled-NOT operation, where $\\mathsf{Y}$ is the control \n",
     "      and $\\mathsf{X}$ is the target:\n",
     "</ol>             \n",
     "            \n",
     "![Second operation e-bit creator](images/3/ebit-circuit-second.png)\n",
     "\n",
     "<ul style=\"list-style-type:none\"><li>        \n",
-    "The controlled-NOT gate's action on standard basis states is as follows:\n",
+    "When the zeroeth qubit is the control and the first qubit is the target, we can represent the controlled-NOT gate by the matrix:\n",
     "</ul>\n",
     "\n",
     "![Controlled-NOT gate](images/3/cNOT.png)\n",
     "        \n",
     "<ul style=\"list-style-type:none\"><li>      \n",
-    "Given that we order the qubits as $(\\mathsf{X},\\mathsf{Y})$, we get that the matrix representation of the controlled-NOT gate is this:\n",
+    "In the image above, the first qubit is the control and the zeroeth is the target, so the controlled-NOT gate has the matrix:\n",
```

Comment:
Same issue as before. An alternative to changing it is to clearly explain the numbering, which we could do above. For example, we could expand the "Ordering of qubits in quantum circuits" remark to "Ordering and numbering qubits in quantum circuits," maybe in a special box of some sort that calls it out. I know Paul is concerned about this issue, and it seems like this is as good a place to explain it as any.